partner_extension_af/views/res_partner_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit.af</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="user_id" position="after">
+                <field name="is_jobseeker" groups="base_user_groups_dafa.1_line_support"/>
```

Comment:
If group `base_user_groups_dafa` used maybe this module shoud be dependent on it?

Reply:
Fixed by: https://github.com/vertelab/odoo-base/pull/222/commits/b9601757dcfde908007243c060f09c928025282c